superdesk/users/services.py

```diff
@@ -363,6 +363,22 @@ def get(self, req, lookup):
                 lookup['is_author'] = bool(int(is_author))
             else:
                 logger.warn('bad value of is_author argument ({value})'.format(value=is_author))
+
+        """filtering out inactive users"""
+
+        params = json.loads(request.args.get('where')) if request and request.args.get('where') else None
+        query = {
+            '$and': [
+                {'is_active': True},
+                {'is_enabled': True},
+            ]
+        }
+
+        if req and not params:
+            req.where = json.dumps(query)
+        elif req and params and params.get('all'):
+            req.where = None
```

Comment:
not sure you should reset this, that might not allow you to search within all users, but just omitting that query should be enough

Reply:
I would not touch a user service at all. We can specify it on the client-side when we query a list of users for marking.

Reply:
the idea was to return only active users by default and those not activated/disabled only when explicitly requested so we don't have to filter the users all over the client